README.md

```diff
@@ -1,15 +1,88 @@
-# Unified Distributed Execution
+<h2 align="center">Unified Distributed Execution</h2>
 
-The framework supports multiple execution backends: Ray, Dask, MPI and MultiProcessing.
+<p align="center">
+<a href="https://unidist.readthedocs.io/en/latest/?badge=latest"><img alt="" src="https://readthedocs.org/projects/unidist/badge/?version=latest" align="center"></a>
+</p>
 
-To run tests you need to install ``pytest`` package:
+### What is unidist?
 
+unidist is a framework that is intended to provide the unified API for distributed execution by supporting various performant execution backends. At the moment the following backends are supported under the hood:
+
+* [Ray](https://docs.ray.io/en/master/index.html)
+* [MPI](https://www.mpi-forum.org/)
+* [Dask Distributed](https://distributed.dask.org/en/latest/)
+* [Python Multiprocessing](https://docs.python.org/3/library/multiprocessing.html)
+
+unidist is designed to work in a [task-based parallel](https://en.wikipedia.org/wiki/Task_parallelism) model.
+
+Also, the framework provides a sequential ``Python`` backend, that can be used for debugging.
+
+### Installation
+
+Currently, unidist can be installed from sources using ``pip``:
+
+```bash
+$ pip install git+https://github.com/modin-project/unidist
```

Comment:
It would be nice to say that in this case only `Python`/`MultiProcessing` backends will work. Or you can add several lines-examples of installation different backend-deps sets like:
```suggestion
$ pip install git+https://github.com/modin-project/unidist  # `MultiProcessing` and `Python` are supported only
$ pip install git+https://github.com/modin-project/unidist#egg=unidist[all]  # all deps
$ pip install git+https://github.com/modin-project/unidist#egg=unidist[ray]  # ray deps
```

Reply:
fixed